harvester/files/sources/anatomy_tool.py

```diff
@@ -0,0 +1,210 @@
+        return [
+            {
+                "mime_type": mime_type,
+                "url": url,
+                "hash": sha1(url.encode("utf-8")).hexdigest(),
+                "title": title,
+                "copyright": default_copyright,
+                "access_rights": "OpenAccess" if default_copyright != "yes" else "RestrictedAccess"
+            }
+            for mime_type, url, title in zip(
+                [mime_node.text.strip() for mime_node in mime_types],
+                [url_node.text.strip() for url_node in urls],
+                [f"URL {ix+1}" for ix, mime_node in enumerate(mime_types)],
+            )
+        ]
+
+    @classmethod
+    def get_url(cls, soup: bs4.BeautifulSoup, file_info: FileInfo) -> str:
+        return file_info.url.text.strip().replace(" ", "+")
+
+    @classmethod
+    def get_hash(cls, soup: bs4.BeautifulSoup, file_info: FileInfo) -> str | None:
+        if not file_info.url:
+            return
+        return sha1(file_info.url.encode("utf-8")).hexdigest()
+
+    @classmethod
+    def get_product_id(cls, soup: bs4.BeautifulSoup, file_info: FileInfo) -> str:
+        return file_info.product.find('identifier').text.strip()
+
+    @classmethod
+    def get_title(cls, soup: bs4.BeautifulSoup, file_info: FileInfo) -> str | None:
+        el = file_info.product
+        node = el.find('title')
+        if node is None:
+            return
+        translation = node.find('string')
+        return translation.text.strip() if translation else None
+
+    @classmethod
+    def get_mime_type(cls, soup: bs4.BeautifulSoup, file_info: FileInfo) -> str | None:
+        el = file_info.product
+        node = el.find('format')
+        if node:
+            return node.text.strip()
+        url = cls.get_url(soup, file_info)
+        if not url:
+            return
+        mime_type, encoding = guess_type(url)
+        return mime_type
+
+    @classmethod
+    def get_is_link(cls, soup: bs4.BeautifulSoup, file_info: FileInfo) -> bool:
+        return file_info.mime_type.text.strip() == "text/html"
+
+    @classmethod
+    def get_copyright(cls, soup: bs4.BeautifulSoup, file_info: FileInfo) -> str:
+        return cls.parse_copyright_description(cls.get_copyright_description(soup, file_info)) or "yes"
+
+    @classmethod
+    def get_provider(cls, soup: bs4.BeautifulSoup, file_info: FileInfo) -> dict[str, None | str]:
+        return {
+            "ror": None,
+            "external_id": None,
+            "slug": "anatomy_tool",
+            "name": "AnatomyTOOL"
+        }
+
+    @classmethod
+    def get_studies(cls, soup: bs4.BeautifulSoup, file_info: FileInfo) -> list[str]:
+        blocks = cls.find_all_classification_blocks(file_info.product, "discipline", "id")
+        return list(set([block.text.strip() for block in blocks]))
+
+    @classmethod
+    def get_access_rights(cls, soup: bs4.BeautifulSoup, file_info: FileInfo) -> None | str:
+        files = cls.get_files(soup, file_info)
```

Comment:
Still used here apparently.
Could become:
```
access_rights_blocks = cls.find_all_classification_blocks(file_info.product, "discipline", "id")
if len(access_rights_blocks):
     return access_rights_blocks[0].text.strip()
return "ClosedAccess"

Reply:
hmmz this doesn't seem to work. It returns an ID instead of the string OpenAccess/RestrictedAccess...